acquia/acquia_cms
# ACMS-988: Update tests Artifacts & Fix Drupal 9.3 deprecated code

**Motivation**
Fixes [ACMS-988](https://backlog.acquia.com/browse/ACMS-988)

**Proposed changes**
- Removed Drupal 9.3 deprecated methods and added alternative.
- Update .travis.yml file to remove PHP 7.3 CI test jobs.
- Updated tests artifacts.
- Removed drupal 9.3 failing patch and we'll fix that in ticket: [ACMS-979](https://backlog.acquia.com/browse/ACMS-979)

**Alternatives considered**
N/A

**Testing steps**
- Install Site.
- Run command: `npm install`.
- Then run command: `npm run backstop-starter` and verify that backstop tests should pass. 

**Merge requirements**
